system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -1120,6 +1120,10 @@ impl pallet_claims::Config for Runtime {
 
 impl pallet_ah_migrator::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	type ManagerOrigin = EitherOfDiverse<
+		EnsureRoot<AccountId>,
```

Comment:
The relay chain will come in as root, right? So we dont need to use this `EnsureXcm`?

Reply:
Right